### ISSUE TITLE
Site fails to build

### ISSUE THREAD

**Issue by josephkerkhof** [sloped/ynab-static#1]
## Description
Installing a fresh clone of this site on my local computer renders build errors when running `npm run build`. My errors are listed below.

```
ERROR #11321  PLUGIN

"gatsby-source-ynab" threw an error while running the sourceNodes lifecycle:

[object Object]
```

I've done some searching to try and resolve the issue and it _seems_ like it could be a package dependency problem or the [`export.sourceNodes`](https://github.com/sloped/ynab-static/blob/6e3cb0ee782fe0d02062c8dee84fddfc56a42683/plugins/gatsby-source-ynab/gatsby-node.js#L4).

Also, my site fails to build in Netlify and fails in the same way.

**Comment by sloped** [comment]
I updated the readme, also included an update that uses Auth0 to do authentication.

**Comment by josephkerkhof** [comment]
I figured out what my problem was. I had to install the `gatsby-source-ynab` plugin manually, which was a _little_ tricky to find because it was in the netlify.toml config file.

Also I had to remove the `goal_target_month` from the graphql. Maybe I just don't have that value in my YNAB API response. I probably don't have goals set up the same way as you.

That said, I've got the site to build, so I'm closing this issue.
